app/views/consent_forms/confirm.html.erb

```diff
@@ -53,4 +53,46 @@
   </div>
 </div>
 
+<h2 class="nhsuk-heading-m">About you</h2>
+
+<div class="nhsuk-card">
+  <div class="nhsuk-card__content">
+    <%= govuk_summary_list classes: 'app-summary-list--no-bottom-border' do |summary_list|
+      summary_list.with_row do |row|
+        row.with_key { "Your name" }
+        row.with_value { @consent_form.parent_name }
+        row.with_action(text: "Change",
+          href: session_consent_form_edit_path(@session, @consent_form, :parent),
+          visually_hidden_text: "your name")
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { "Relationship" }
+        row.with_value { @consent_form
+          .human_enum_name(:parent_relationship)
+          .capitalize }
+        row.with_action(text: "Change",
+          href: session_consent_form_edit_path(@session, @consent_form, :parent),
+          visually_hidden_text: "your relationship")
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { "Email address" }
+        row.with_value { @consent_form.parent_email }
+        row.with_action(text: "Change",
+          href: session_consent_form_edit_path(@session, @consent_form, :parent),
+          visually_hidden_text: "your email")
+      end
+
+      summary_list.with_row do |row|
+        row.with_key { "Phone number" }
+        row.with_value { @consent_form.parent_phone }
+        row.with_action(text: "Change",
+          href: session_consent_form_edit_path(@session, @consent_form, :parent),
+          visually_hidden_text: "your phone")
+      end
+    end %>
```

Comment:
This isn't the exhaustive list; notably the `_other` stuff isn't visible. I can add it now, or we can come back to it later, the prototype isn't up to date on how this should look in every permutation.

Reply:
Is this something that we'll easily catch in a snagging session, or should we card it up?

Reply:
Let's leave it for later...